anti-diff/src/AntiDiff/Util/Auto.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE DerivingStrategies         #-}
```

Comment:
Do we use this?

Reply:
I'm not sure if you refer to the module or language extension -- If you refer to the module, then yes it is being used in tests for (semi-)group(-oid) laws